pages/contact/templates/contact/contact_form.html

```diff
@@ -0,0 +1,131 @@
+      {% endif %}
+      <p class="mt-1 text-sm text-gray-500">Please provide your e-mail address in case you would like to receive a response from us. In other cases, the e-mail address is not required.</p>
+    </div>
+
+    <div>
+      <div class="block text-sm font-medium text-gray-900">What kind of a question or suggestion do you have?
+        <span class="text-red-600">*</span></div>
+      <div class="mt-2 space-y-2">
+        <label class="flex items-center gap-2 text-gray-900">
+          <input type="checkbox" name="suggestion" id="id_suggestion" {% if form.suggestion.value %}checked{% endif %} class="h-4 w-4 rounded border-gray-300 text-teal-600 focus:ring-teal-500">
+          <span>Suggestion for the Portal</span>
+        </label>
+        <label class="flex items-center gap-2 text-gray-900">
+          <input type="checkbox" name="dm_support" id="id_dm_support" {% if form.dm_support.value %}checked{% endif %} class="h-4 w-4 rounded border-gray-300 text-teal-600 focus:ring-teal-500">
+          <span>Request for help with data management or data sharing questions</span>
+        </label>
+        <label class="flex items-center gap-2 text-gray-900">
+          <input type="checkbox" name="other" id="id_other" {% if form.other.value %}checked{% endif %} class="h-4 w-4 rounded border-gray-300 text-teal-600 focus:ring-teal-500">
+          <span>Other</span>
+        </label>
+      </div>
+      {% if form.non_field_errors %}
+        <p class="mt-1 text-sm text-red-600">Please select at least one alternative.</p>
+      {% endif %}
+      <p class="mt-1 text-sm text-gray-500">You can ask a question or suggest one or more items using this form.</p>
+    </div>
+
+    <div>
+      <label for="id_message" class="block text-sm font-medium text-gray-900">Message</label>
+      <textarea name="message" id="id_message"
+                class="mt-1 block w-full rounded-md border {% if form.message.errors %}border-red-600{% else %}border-gray-300{% endif %} px-3 py-2 shadow-sm focus:outline-none focus:ring-2 focus:ring-teal-500 min-h-40"
+                {% if form.message.errors %}aria-invalid="true" aria-describedby="id_message_error" autofocus{% endif %}
+                required minlength="20" maxlength="5000">{{ form.message.value|default_if_none:'' }}</textarea>
+      {% if form.message.errors %}
+        <p id="id_message_error" class="mt-1 text-sm text-red-600">{{ form.message.errors.0 }}</p>
+      {% endif %}
+      <p class="mt-1 text-sm text-gray-500">Please describe what you need help with (e.g., sharing your own dataset) or
+        provide link(s) and/or DOI(s) to your suggested dataset, project, publication, preprint, etc. and anything else
+        that we should know.</p>
+    </div>
+
+    <div>
+      <button type="submit" class="inline-flex items-center rounded-md bg-teal-600 px-4 py-2 text-white hover:bg-indigo-500 focus:outline-none focus:ring-2 focus:ring-indigo-500">Send message</button>
```

Comment:
I know we are not finalising styles now, but wanna mention as this is small and I might forget later.

`bg-indigo-500`, keep it within portal's visual identity or at least closer colours. This is very different and new.